lib/galaxy/tool_util/linters/tests.py

```diff
@@ -15,7 +15,9 @@ def lint_tsts(tool_xml, lint_ctx):
     num_valid_tests = 0
     for test in tests:
         has_test = False
-        if "expect_failure" in test.attrib or "expect_exit_code" in test.attrib:
+        if "expect_failure" in test.attrib \
```

Comment:
That doesn't look like an improvement. If you want to change this how about creating a set outside of the loop `test_expect = {{'expect_failure', 'expect_exit_code', 'expenct_num_outputs'}}`
and then`if test_expect.intersection(set(test.attrib)):` here ?